ArduCopter/land_detector.cpp

```diff
@@ -70,9 +71,16 @@ void Copter::update_land_detector()
         const bool landing = flightmode->is_landing();
         bool motor_at_lower_limit = (flightmode->has_manual_throttle() && (motors->get_below_land_min_coll() || heli_flags.coll_stk_low) && fabsf(ahrs.get_roll()) < M_PI/2.0f)
                                     || ((!force_flying || landing) && motors->limit.throttle_lower && pos_control->get_vel_desired_cms().z < 0.0f);
+        bool mix_ready_for_landing = true;
 #else
         // check that the average throttle output is near minimum (less than 12.5% hover throttle)
-        bool motor_at_lower_limit = motors->limit.throttle_lower && attitude_control->is_throttle_mix_min();
+        bool motor_at_lower_limit = motors->limit.throttle_lower;
+        bool mix_ready_for_landing = attitude_control->is_throttle_mix_min();
```

Comment:
mix_ready_for_landing - This name should describe the state. "throttle_mix_at_min" for example.

Reply:
Thanks fixed